packages/components/src/__alpha__/SingleSelect/SingleSelect.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { type HTMLAttributes, type PropsWithChildren } from 'react'
+import {
+  Label as RACLabel,
+  Popover as RACPopover,
+  Select as RACSelect,
+} from 'react-aria-components'
+import { FieldGroup } from '~components/FieldGroup'
+import { FieldMessage } from '~components/FieldMessage'
+import { type InputStatusType } from '~components/Input'
+import { Label } from '~components/Label'
+import { type OverrideClassName } from '~components/types/OverrideClassName'
+import { List, ListItem, ListSection, PopoverContent, Trigger } from './subcomponents'
+import styles from './SingleSelect.module.scss'
+
+export type Item = {
+  label: string
+  value: string
+  disabled?: boolean
+}
+
+export type ItemGroup = {
+  label: string
+  options: Item[]
+}
+
+export type Items = {
+  items?: ItemGroup[] | Item[]
+}
+
+export type SingleSelectProps = {
```

Comment:
I would pare this component back to the basics and drop the labelText, inline, validationMessage etc we want to just introduce the basic folder structure and not anything that isn't intentional to this set PR